ingestor/chalicelib/yankee.py

```diff
@@ -0,0 +1,414 @@
+from typing import Dict, Optional, Tuple
+import requests
+import time
+import pandas as pd
+import json
+import boto3
+from keys import YANKEE_API_KEY
+from botocore.exceptions import ClientError
+from datetime import datetime
+from tempfile import TemporaryDirectory
+from geopy import distance
+import s3
+from ddtrace import tracer
+from dataclasses import dataclass
+from chalicelib import dynamo
+
+from typing import List
+from sqlalchemy.orm import Session
+from mbta_gtfs_sqlite import MbtaGtfsArchive
+from mbta_gtfs_sqlite.models import (
+    RoutePattern,
+    RoutePatternTypicality,
+    Trip,
+    ShapePoint,
+    Stop
+)
+
+ShapeDict = Dict[str, List[ShapePoint]]
+Coords = Tuple[float, float]
+
+BUCKET = "tm-shuttle-positions"
+KEY = "yankee/last_shuttle_positions.csv"
+BOSTON_COORDS = (-71.057083, 42.361145)
+OSRM_DISTANCE_API = "http://router.project-osrm.org/route/v1/driving/"
+METERS_PER_MILE = 0.000621371
+SHUTTLE_PREFIX = "Shuttle"
+STOP_RADIUS_MILES = 0.1
+TIME_FORMAT = "%Y-%m-%d-%H:%M:%S"
+SHUTTLE_TRAVELTIME_TABLE = "ShuttleTravelTimes"
```

Comment:
My initial thought is that we make a new table for storing shuttle travel times, since it's not really aggregate data like what we store in `DeliveredTripMetrics`. Happy to hear any suggestions here though